django/db/models/base.py

```diff
@@ -39,6 +36,12 @@
 from django.utils.translation import ugettext_lazy as _
 from django.utils.version import get_version
 
+DEFERRED = object()
```

Comment:
I wonder if making `DEFERRED` an object with a custom `__str__` implementation might help at all with debugging in case anyone has overridden `__init__` and is trying to figure out what these objects are that they are now getting as values for the deferred fields?